gen/installer/azure.py

```diff
@@ -150,8 +149,7 @@ def gen_templates(user_args, arm_template):
         results.templates[arm_template + '.json'],
         variant_cloudconfig['master'],
         variant_cloudconfig['slave'],
-        variant_cloudconfig['slave_public']
-        )
+        variant_cloudconfig['slave_public'])
```

Comment:
>  only one of those two can be allowed at a time. 

OK. If I must choose between the last two, I would opt for the latter:
```
foo = bar(
    boo,
    far
    )
```
But this is a small dilemma:

I really tend to prefer that, because I believe it visually integrates better with Python's overall indentation concept. Using this style is my year-old habit, and if we were to enforce "closing bracket to the far left" (your 2nd code block), I would _constantly_ run against that wall.

However, what I described is just my personal style preference. I entirely get that people coming from languages with curly brackets like and tend to do what your 2nd code block shows.

Having said this, I will be "OK" with either way here.